tests/reg_tests/test_adjoint.py

```diff
@@ -326,6 +407,12 @@ def cmplx_test_geom_dvs(self):
 
         xRef = {"twist": [0.0] * 6, "span": [0.0], "shape": numpy.zeros(72, dtype="D")}
 
+        if self.name == "Rotating_wing":
+            rtol = 4e-6
+        else:
+            rtol = 1e-8
+        atol = 5e-10
```

Comment:
Are these the tightest possible tolerances for the non-rotating cases? We were using rtol of 5e-9 before, but that was with a different atol.

Reply:
oh yeah my bad, I copy-pasted the tolerances from the previous test.